Makefile

```diff
@@ -1479,6 +1480,12 @@ llama-cvector-generator: examples/cvector-generator/cvector-generator.cpp \
 	$(CXX) $(CXXFLAGS) -c $< -o $(call GET_OBJ_FILE, $<)
 	$(CXX) $(CXXFLAGS) $(filter-out %.h $<,$^) $(call GET_OBJ_FILE, $<) -o $@ $(LDFLAGS)
 
+# TODO: Move to tests
```

Comment:
I would love to move this to `tests` too. Let me know if you need help to do that.

Reply:
Nice! I might need some help since I’m not too familiar with the llama.cpp tests. It would be great to automate a comparison with scipy outputs tho. I’ll take care of the other fixes first, and maybe we can address this afterward ?